projects/indicators/src/modules/chevron/chevron.component.ts

```diff
@@ -6,15 +6,40 @@ import { Component, EventEmitter, Input, Output } from '@angular/core';
   templateUrl: './chevron.component.html',
 })
 export class SkyChevronComponent {
-  @Output()
-  public directionChange = new EventEmitter<string>();
+  @Input()
+  public ariaControls: string;
 
   @Input()
-  public direction = 'up';
+  public ariaLabel: string;
+
+  @Input()
```

Comment:
Does this need a JSDocs comment to add the public property to the docs? I assume so. I'm not absolutely sure what we need to say here, but I assume it's something along the lines of:

```Specifies whether the chevron points up or down.```